05_local-governance/descriptive-representation/descriptive_representation_denom.qmd

```diff
@@ -70,34 +78,45 @@ myvars <- c(
   "B03002_008E",
   "B03002_009E"
   )
+```
+
+
+Pull ACS demographic data at the Census Place and Census County levels.
 
-# Pull ACS data at the Census Place and Census County levels
+```{r}
 places_demo <- get_acs(geography = "place",
                        variables = myvars,
```

Comment:
There's 2022 acs data - just want to double check that using 2021 is intentional